operators/multiclusterobservability/manifests/base/multicluster-observability-addon/cluster_role.yaml

```
@@ -0,0 +1,70 @@
+  kind: ClusterRole
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    labels:
+      app.kubernetes.io/component: multicluster-observability-addon-manager
+      app.kubernetes.io/name: multicluster-observability-addon-manager
+      app.kubernetes.io/part-of: multicluster-observability-addon
+    name: multicluster-observability-addon-manager
+  rules:
+    - apiGroups: [""]
+      resources: ["configmaps", "events"]
+      verbs: ["get", "list", "watch", "create", "update", "delete", "deletecollection", "patch"]
+    - apiGroups: ["coordination.k8s.io"]
+      resources: ["leases"]
+      verbs: ["get", "list", "watch", "create", "update", "patch"]
+    - apiGroups: ["authorization.k8s.io"]
+      resources: ["subjectaccessreviews"]
+      verbs: ["get", "create"]
+    - apiGroups: ["cluster.open-cluster-management.io"]
+      resources: ["managedclusters", "placements", "placementdecisions"]
+      verbs: ["get", "list", "watch"]
+    - apiGroups: ["addon.open-cluster-management.io"]
+      resources: ["managedclusteraddons/finalizers"]
+      verbs: ["update"]
+    - apiGroups: [ "addon.open-cluster-management.io" ]
+      resources: [ "clustermanagementaddons/finalizers" ]
+      verbs: [ "update" ]
+    - apiGroups: [ "addon.open-cluster-management.io" ]
+      resources: [ "clustermanagementaddons/status" ]
+      verbs: ["update", "patch"]
+    - apiGroups: ["addon.open-cluster-management.io"]
+      resources: ["clustermanagementaddons"]
+      verbs: ["get", "list", "watch"]
+    - apiGroups: ["addon.open-cluster-management.io"]
+      resources: ["managedclusteraddons"]
+      verbs: ["get", "list", "watch", "create", "update", "delete"]
+    - apiGroups: ["addon.open-cluster-management.io"]
+      resources: ["managedclusteraddons/status"]
+      verbs: ["update", "patch"]
+    - apiGroups: ["addon.open-cluster-management.io"]
+      resources: ["addontemplates", "addondeploymentconfigs"]
+      verbs: ["get", "list", "watch"]
+    - apiGroups: ["work.open-cluster-management.io"]
+      resources: ["manifestworks"]
+      verbs: ["get", "list", "watch", "create", "update", "delete", "patch"]
+    # addon template controller needs these permissions to approve CSR
+    - apiGroups: ["certificates.k8s.io"]
+      resources: ["certificatesigningrequests"]
+      verbs: ["create", "get", "list", "watch"]
+    - apiGroups: ["certificates.k8s.io"]
+      resources: ["certificatesigningrequests/approval", "certificatesigningrequests/status"]
+      verbs: ["update"]
+    - apiGroups: ["certificates.k8s.io"]
+      resources: ["signers"]
+      verbs: ["approve", "sign"]
+    - apiGroups: ["rbac.authorization.k8s.io"]
+      resources: ["rolebindings"]
+      verbs: ["get", "list", "watch", "create", "delete"]
+    # The addon will need to create secrets to ensure secure communication
+    - apiGroups: [""]
+      resources: ["secrets"]
+      verbs: ["get", "list", "watch", "create", "update", "delete", "patch"]
```

Just noticed we also don't need these 
```suggestion
      verbs: ["get", "list", "watch"]
```

---

https://github.com/rhobs/multicluster-observability-addon/pull/57